Resources/FAQ/Jobs-Discord-Asked-Questions.md

```diff
@@ -252,13 +308,13 @@ https://Github.com/Zrips/Jobs/wiki/Commands
 
 > Hi, is there any way to be able to open the jobs menu using /jobs and not /jobs browse
 
-Not by default, another plugin might be required to alias the /jobs main command, making sure that it goes to /jobs browse, or /jobs help. It would be nice ot have a toggle. 
+Not by default, another plugin might be required to alias the /jobs command, ensuring that it opens /jobs browse instead of /jobs help. It would be nice to have a toggle. 
```

Comment:
Suggestion, not introduced by you
```suggestion
Not by default, another plugin might be required to alias the `/jobs` command, ensuring that it opens `/jobs browse` instead of `/jobs help`. It would be nice to have a toggle. 
```